### DISCUSSION TITLE
AttributeError: 'NoneType' object has no attribute 'arch'

### DISCUSSION THREAD

**Discussion by eneko98** [facebookresearch/esm#272] [General]
I am trying to train an ESM-2 model but I get the following error:

Traceback (most recent call last):
  File "ESM-1b.py", line 52, in <module>
    model, alphabet = esm.pretrained.load_model_and_alphabet_local("/gpfs/projects/bsc08/bsc08936/ESM/models/esm2_t30_150M_UR50D.pt")
  File "/home/bsc08/bsc08936/.local/lib/python3.6/site-packages/esm/pretrained.py", line 68, in load_model_and_alphabet_local
    return load_model_and_alphabet_core(model_data, regression_data)
  File "/home/bsc08/bsc08936/.local/lib/python3.6/site-packages/esm/pretrained.py", line 81, in load_model_and_alphabet_core
    alphabet = esm.Alphabet.from_architecture(model_data["args"].arch)
AttributeError: 'NoneType' object has no attribute 'arch'

I am training it on a cluster so I had to download the model "esm2_t30_150M_UR50D.pt" locally because I have no internet. Does anyone know what's wrong?

**Comment by tomsercu** [comment]
From #255 , could this be the same issue? 
> It seems like you still have old esm package installed so it was imported instead of the current version, please remove it via "pip uninstall fair-esm" and install the current one with e.g. "pip install -e ."
